src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -514,14 +514,16 @@ class SchedulerActions(
     * Make sure the app is running the correct number of instances
     */
   def scale(driver: SchedulerDriver, app: AppDefinition): Unit = {
-    // FIXME (217): use taskTracker.countRunning
-    val currentCount = taskTracker.appTasksSync(app.id).count(_.launched.isDefined)
+    val tasks = taskTracker.appTasksSync(app.id)
+    val runningCount = tasks.count(_.launched.isDefined)
+    val reserved = tasks.count(t => t.reservationWithVolumes.isDefined && t.launched.isEmpty)
+    val runningReserved = runningCount + reserved
     val targetCount = app.instances
 
-    if (targetCount > currentCount) {
-      log.info(s"Need to scale ${app.id} from $currentCount up to $targetCount instances")
+    if (targetCount > runningReserved) {
```

Comment:
As discussed: maybe best if you revert these changes and we keep evaluating the taskTracker info based on `_.launched.isDefined ~ running` and check the launchQueue afterwards.